### PR TITLE
Feature/sponsors

### PR DESCRIPTION
**Pull request by maryharebear**
Problems
- media queries

Improvements
- animation
- more logos?
[Sponsors Component](https://app.gitkraken.com/glo/board/XYAsE7iuYQAP_TPG/card/XYAsFOD5fAAPcwCi)

### CODE REVIEW

**Comment by shiv248** [comment]
Feel free to reply to the conversation to keep a discussion going :)

File: client/src/component/Sponsors/index.css [hunk 1]
```diff
@@ -0,0 +1,745 @@
...
+    vertical-align: middle;
+    align-items: center;
+    bottom: 100px;
+    position: relative;
+    left: 80px;
+    width: 100%;
+    max-width: 280px;
+    height: 125px;
+}
+body {
```
[the remark this answers is not in the document]
Comment (maryharebear) [reply client/src/component/Sponsors/index.css]: What should I change it to? in form of the background image? and after I make all the changes do I make another pull request?

[the remark this answers is not in the document]
Comment (shiv248) [reply client/src/component/Sponsors/index.css]: I'd recommend this change:
```suggestion
App-sponsors {
   background-color:#2a2834 
}
```
also change the respective `class` or `className` to make them synced

After you finish your changes you just push again, and then at the bottom @ me "@shiv248"  and a message of you completing the changes and also if you had any issues or did it differently.

**Comment by shiv248** [comment]
@maryharebear Can you delete the Title component folder in your branch, this is so that it is easier to merge branches.
image

Also, don't forget to delete the import and `< Title />`
image

File: client/src/component/Sponsors/sponsors-logos.jsx [hunk 2]
```diff
@@ -22,31 +26,55 @@ class Sponsors extends React.Component {
...
+                <div className= "3row">
             <a href = "https://www.sketch.com/"> {/*goes to sketch site*/}
                  <img className = "sketch" src = {sketch} alt="Sketch" /> {/*Sketch logo and link*/}
             </a>
-                         </div>
+               
+                 <div className = "bottom">
+                <a href = "https://balsamiq.com/">
+                        <img className = "balsamiq" src = {balsamiq} alt= "Balsamiq" />
+            </a>
```
Comment (shiv248) [inline client/src/component/Sponsors/sponsors-logos.jsx]: image
This looks scrunched up. do you think you can find a better image for this company?

Comment (maryharebear) [reply client/src/component/Sponsors/sponsors-logos.jsx]: sorry, I will just adjust the width size

**Comment by adriandarian** [comment]
@all-contributors please add @maryharebear for code

**Pull request merged**